objectRecognitionSource/src/main/java/edu/ml/tensorflow/ObjectDetector.java

```diff
@@ -56,20 +57,23 @@
     /**
      * Initializes the ObjectDetector with the given graph and and labels.
      * <br>Edited to initialize and save the graph for reuse, and allow the files to be specified dynamically.
-     * @param graphFile The location of a proto buffer file describing the YOLO net 
-     * @param labelFile The location of the labels for the given net.
-     * @param imageUtil The instance of the ImageUtil class used to save images. Either initialized, or set to null.
-     * @param outputDir The directory to which images will be saved.
-     * @param saveRate  The rate at which images will be saved, once per every saveRate frames. Non-positive values are
-     *                  functionally equivalent to 1. If outputDir is null does nothing.
-     * @param vantiq    The Vantiq variable used to connect to the VANTIQ SDK. Either authenticated, or set to null.
+     * @param graphFile     The location of a proto buffer file describing the YOLO net 
+     * @param labelFile     The location of the labels for the given net.
+     * @param imageUtil     The instance of the ImageUtil class used to save images. Either initialized, or set to null.
+     * @param outputDir     The directory to which images will be saved.
+     * @param labelImage    The boolean flag signifying if images should be saved with or without bounding boxes. If true,
+     *                      the frames will be saved with bounding boxes, and vice versa.     
+     * @param saveRate      The rate at which images will be saved, once per every saveRate frames. Non-positive values are
+     *                      functionally equivalent to 1. If outputDir is null does nothing.
+     * @param vantiq        The Vantiq variable used to connect to the VANTIQ SDK. Either authenticated, or set to null.
      */
-    public ObjectDetector(float thresh, String graphFile, String labelFile, ImageUtil imageUtil, String outputDir, int saveRate, Vantiq vantiq) {
+    public ObjectDetector(float thresh, String graphFile, String labelFile, ImageUtil imageUtil, String outputDir, Boolean labelImage, int saveRate, Vantiq vantiq) {
```

Comment:
Another "we should change" at the next chance.

We shouldn't use the presence of the imageUtil class as a boolean. Instead, make this detection explicit.

There seem to be a few similar things (if the VANTIQ parameter is set, use it otherwise...). 

I'm assuming there are more than one ImageUtil subclasses, one for VANTIQ, another for files?

It's a matter of style, but generally overloading meanings makes things more confusing.